docs-chef-io/content/server/v14_0/config_rb_server_optional_settings.md

```diff
@@ -106,6 +106,14 @@ This configuration file has the following general settings:
 :   The topology of the Chef Infra Server. Possible values: `manual`,
     `standalone`, and `tier`. Default value: `'standalone'`.
 
+`opscode_erchef["include_version_in_status"]`
+
+:   Set to `true` if you want to include `server_version` as part of the `/_status` endpoint. By default the value is set to `false`.
```

Comment:
```suggestion
:   Set to `true` if you want to include `server_version` as part of the `/_status` endpoint. Default value: `false`.
```

Reply:
Suggestion: "Set to `true` to use `server_version` as part of the `/_status` endpoint."